test/unit/utils.spec.js

```diff
@@ -288,6 +288,44 @@ describe('lib/utils', function () {
         ].join('\n');
         expect(stringify(expected), 'to be', actual);
       });
+
+      describe('should represent null prototypes', function () {
+        it('With explicit names', function () {
+          const foo = Object.create(null, {
+            [Symbol.toStringTag]: {value: 'Foo'},
+            bing: {get: () => 'bong', enumerable: true}
+          });
+          const expected = [
+            '{',
+            '  "[Symbol.toStringTag]": "Foo"',
```

Comment:
If we wanted to either:

1. Always expose the `toStringTag` if present
2. Never expose the `toStringTag`

Then we could just return `object` instead of `null-prototype` and the logic would work.

Only really need a special case if we are conditionally exposing it...

Reply:
I don't have a strong preference either way. What's working now fixes the issue and is nicely informative on my end.